rules/decoder-events.rules

```diff
@@ -71,6 +71,7 @@ alert pkthdr any any -> any any (msg:"SURICATA UDP header length too small"; dec
 alert pkthdr any any -> any any (msg:"SURICATA UDP invalid length field in the header"; decode-event:udp.len_invalid; classtype:protocol-command-decode; sid:2200120; rev:2;)
 alert pkthdr any any -> any any (msg:"SURICATA SLL packet too small"; decode-event:sll.pkt_too_small; classtype:protocol-command-decode; sid:2200041; rev:2;)
 alert pkthdr any any -> any any (msg:"SURICATA Ethernet packet too small"; decode-event:ethernet.pkt_too_small; classtype:protocol-command-decode; sid:2200042; rev:2;)
+alert pkthdr any any -> any any (msg:"SURICATA Ehtertype Unknown"; decode-event:ethernet.unknown_ethertype; threshold: type limit, track by_rule, seconds 60, count 1; classtype:protocol-command-decode; sid:2200121; rev:1;)
```

Comment:
Is there also a similar rule for ip type unknown ?

Reply:
unrelated to Philippe's question: seeing a typo here Ehternet should be Ethernet

Reply:
> Is there also a similar rule for ip type unknown ?

There's several events relating to unknowns with IPv4. Which are you referring to -- when the IP packet contains an unknown protocol?

Reply:
> unrelated to Philippe's question: seeing a typo here Ehternet should be Ethernet

Will fix

Reply:
> when the IP packet contains an unknown protocol?

Yes, I meant that

`    switch (p->proto) {`has no default case in `DecodeIPV4`

Reply:
It's good to track this in a ticket, its a good point but mostly off topic for this PR.

Reply:
This was a part of the demand I had forgotten :

https://redmine.openinfosecfoundation.org/issues/7146